### PR TITLE
Fix QrCode methods in minified version

### PR DESCRIPTION
**Pull request by aaaronlopez**
cc @BranchMetrics/core-team 

QrCodes.rawBuffer would be undefined because of minification

### CODE REVIEW

File: src/6_branch.js [hunk 1]
```diff
@@ -1488,14 +1488,16 @@ Branch.prototype['qrCode'] = wrap(
...
-				QrCode.rawBuffer = rawBuffer;
-				QrCode.base64 = function() {
-					// First Encode array buffer as UTF-8 String, then Base64 Encode
-					if (this.rawBuffer) {
-						return btoa(String.fromCharCode.apply(null, new Uint8Array(this.rawBuffer)));
-					}
-					throw Error('QrCode.rawBuffer is empty.');
-				};
+				if (!error) {
+					QrCode['rawBuffer'] = rawBuffer;
```
Comment (aaaronlopez) [inline src/6_branch.js:1492]: these have to be explicitly called out as strings, otherwise the keys may get minified

**Comment by aaaronlopez** [comment]
pip previously failed with 
```
deployment/deploy.sh: line 175: pip: command not found

Exited with code exit status 127
CircleCI received exit code 127
```

**Comment by aaaronlopez** [comment]
@jleebranch @jdee bump for review when you have the chance

**Review by jdee — APPROVED** [review]
:+1:

**Pull request merged**